common/lib/xmodule/xmodule/modulestore/tests/test_publish.py

```diff
@@ -1,16 +1,36 @@
 """
 Test the publish code (mostly testing that publishing doesn't result in orphans)
 """
+import unittest
+import ddt
+import itertools
+from shutil import rmtree
+from tempfile import mkdtemp
 from nose.plugins.attrib import attr
+from nose.tools import (                        # pylint: disable=W0611,E0611
+    assert_true, assert_false,
+    assert_equals, assert_not_equals,
+    assert_is, assert_is_not,
+    assert_is_instance, assert_is_none,
+    assert_in, assert_not_in,
+    assert_raises, assert_raises_regexp,
```

Comment:
I'm not actually deriving from `TestCase` anywhere currently. Not sure if that plays well with the trickery at the bottom of this file.